release-notes/7.0/known-issues.md

```diff
@@ -16,6 +16,24 @@ More information and workaround can be found at https://github.com/dotnet/runtim
 
 ## .NET SDK
 
+### [Unhandled Exception in dotnet format app in .NET 7.0 Preview 5](https://github.com/dotnet/sdk/issues/25879)
+
+dotnet format app that comes with SDK has this exception:
+Unhandled exception: System.IO.FileLoadException: Could not load file or assembly 'System.Configuration.ConfigurationManager, Version=6.0.0.0
+
+Workaround:
+
+[Install dotnet-format as a global tool](https://github.com/dotnet/format#how-to-install-development-builds)
+
+`dotnet tool install -g dotnet-format --version "7.*" --add-source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7/nuget/v3/index.json`
+
+Then invoke the global tool instead of using the dotnet CLI.
```

Comment:
The part that trips up most users who want to run the global tool is the need for the hyphen. It may make sense to call this out.

```suggestion
Then invoke the global tool using `dotnet-format` instead of through the dotnet CLI using `dotnet format`.
```